File: code/c/storing-using-data/assignment-test.c
#include "splashkit.h"

/* Program assignment_tests.c
* Demonstrates assignment to a variable. 
*/
int main()
{
    int my_data = 10, days_in_month, days_remaining;
    write("my_data is ");
    write_line(my_data);
    
    my_data = my_data + 1;    //add 1 to my_data and store in my_data
    write("my_data is ");
    write_line(my_data);
    
    my_data++;    //add 1 to my_data and store in my_data
    write("my_data is ");
    write_line(my_data);
    
    my_data *= 2;    //double my_data and store in my_data
    printf("my_data is %d\n", my_data);
    
    days_in_month = 365 / 12; //assign days_in_month a calculated value 
    printf("On average there are " + to_string(days_in_month) + " days in a month.");  
    
    //assign days_remaining a calculated value
    days_remaining = 365 - days_in_month * 12;
    write_line("The remaining " + to_string(days_remaining) + " days are distributed to a few months.",
        days_remaining);
    
    return 0;
}

File: code/pascal/storing-using-data/AssignmentTest.pas
program AssignmentTest;

procedure Main();
var
    myData: Integer = 10;
    daysInMonth, daysRemaining: Integer;
begin
    WriteLn('myData is ', myData);
    
    myData := myData + 1;    //add 1 to myData and store in myData
    WriteLn('myData is ', myData);  
    
    myData += 1;    //add 1 to myData and store in myData
    WriteLn('myData is ', myData);  
    
    myData *= 2;    //double myData and store in myData
    WriteLn('myData is ', myData);
    
    daysInMonth := 365 div 12; //assign daysInMonth a calculated value 
    WriteLn('On average there are ', daysInMonth, ' days in a month.');  
    
    //assign daysRemaining a calculated value
    daysRemaining := 365 - daysInMonth * 12;
    WriteLn('The remaining ', daysRemaining, 
            'days are distributed to a few months.');
end;

begin
    Main();
end.